python_modules/libraries/dagster-airbyte/dagster_airbyte/asset_defs.py

```diff
@@ -114,6 +115,7 @@ def _build_airbyte_asset_defn_metadata(
             "destination_tables": destination_tables,
             "normalization_tables": normalization_tables,
             "io_manager_key": io_manager_key,
+            "freshness_policy": freshness_policy,
```

Comment:
I was somewhat surprised that this survives the json serialization check we do, but turns out namedtuples just work. nice